.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10.0-beta.3']
```

Comment:
Don't you want to test on 3.10 rc 1 ? Since all the match rules are only tested on 3.10, it seemed better to me to test on 3.10 also.

Reply:
Yeah, I plan to submit a new PR once the RC is available to test

Reply:
I believe it is already: https://github.com/actions/python-versions/releases